lib/registrant.rb

```diff
@@ -0,0 +1,22 @@
+class Registrant
+  attr_reader :name, :age, :permit_status, :license_data
+
+  def initialize(name, age, permit_status = false)
+    @name = name
+    @age = age
+    @permit_status = permit_status
+    @license_data = {
+      :written => false,
+      :license => false,
+      :renewed => false
```

Comment:
```suggestion
      written: false,
      license: false,
      renewed: false
```

This syntax is a bit more conventional for ruby hashes